doc/nrf/libraries/networking/aws_iot.rst

```diff
@@ -98,18 +107,21 @@ To enable this feature, set the ``client_id`` entry in the :c:struct:`aws_iot_co
    To enable the use of the cloud, API set the configurable option :kconfig:option:`CONFIG_CLOUD_API`, in addition to the other selected library options.
 
 Initializing the library
```

Comment:
Line 98, since it just one Kconfig option, we could combine it with the sentence and avoid the bullet point.